reference/channels.v3.yml

```
@@ -1545,8 +1548,8 @@ components:
   examples:
     create_channel_req_example:
       value:
-        name: Facebook by Meta
-        platform: facebook by meta
+        name: eBay
+        platform: eBay
```

```suggestion
        platform: ebay
```
In the intro, all the platforms are lowercase. It might not make a difference in reality, but it can't hurt for them to match.